x/wasm/client/cli/query.go

```diff
@@ -35,10 +36,30 @@ func GetQueryCmd() *cobra.Command {
 		GetCmdGetContractHistory(),
 		GetCmdGetContractState(),
 		GetCmdListPinnedCode(),
+		GetCmdVMVersion(),
 	)
 	return queryCmd
 }
 
+// GetCmdVMVersion gets current libwasmvm version.
+func GetCmdVMVersion() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "vm-version",
```

Comment:
The term "VM" is a bit overused and `wasmvm` is the Go project that contains code for both sides of the FFI interface. To avoid confusion, I'd go with "libwasmvm-version". A better abbreviation would be "lib-version". If we need something short, let's use that as alias.

Reply:
sounds good to me

Reply:
renamed and added the alias as well.